themes/bootstrap3/templates/search/versions-searchbox.phtml

```diff
@@ -1,24 +1 @@
-<?php
-    $options = $this->searchOptions($this->searchClassId);
-    $searchHome = $options->getSearchHomeAction();
-    $advSearch = $options->getAdvancedSearchAction();
-
-    $hiddenFilters = $this->searchTabs()->getHiddenFilters($this->searchClassId, false, false);
-    if (empty($hiddenFilters)) {
-        $hiddenFilters = $this->searchMemory()->getLastHiddenFilters($this->searchClassId);
-        if (empty($hiddenFilters)) {
-            $hiddenFilters = $this->searchTabs()->getHiddenFilters($this->searchClassId);
-        }
-    }
-    $hiddenFilterParams = $this->searchTabs()->getCurrentHiddenFilterParams($this->searchClassId, false, '?');
-?>
-<?php $tabConfig = $this->searchTabs()->getTabConfig($this->searchClassId, '', $this->searchIndex, 'basic', $hiddenFilters); ?>
-<?php $tabs = $this->context($this)->renderInContext('search/searchTabs', ['searchTabs' => $tabConfig['tabs']]); ?>
-<?php if (!empty($tabs)): ?><?=$tabs ?><div class="tab-content clearfix"><?php endif; ?>
-  <p class="adv_search_links">
-    <a href="<?=$this->url($searchHome) . $hiddenFilterParams?>"><?=$this->transEsc('Start a new Basic Search')?></a>
-    <?php if ($advSearch): ?>
-      | <a href="<?=$this->url($advSearch) . $hiddenFilterParams?>"><?=$this->transEsc('Start a new Advanced Search')?></a>
-    <?php endif; ?>
-  </p>
-<?php if (!empty($tabs)): ?></div><?php endif; ?>
+<?=$this->render('search/no-input-searchbox.phtml');
```

Comment:
Would it be better to just link directly to no-input-searchbox instead of aliasing from versions-searchbox? Or do you think there might be a compelling reason to customize this independently?

Reply:
I wanted to leave the option to easily customize it, and to not cause another back-compatibility break.